src/components/Product/StyledProduct.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+
+const ProductContainer = styled.div`
+	background-color: white;
+	max-width: 384px;
+	max-height: 436px;
+	padding: 20px 0 20px 20px;
+	display: flex;
+	border-radius: 50px;
+`;
+
+const ImageWrapper = styled.div`
+	width: 250%;
+	height: 386px;
+	position: relative;
+	object-fit: cover;
+	border-radius: 50px;
+	overflow: hidden;
+	border: 3px solid #4b587f;
+`;
+
+const ProductArticle = styled.div`
```

Comment:
And I have a question: In my opinion it is not necessary to have `Product` as part of the name of every styled-component for your `Product` component. Did you decide this in your course that it is useful? If not I would recommend to only work with `Article`, `Container` etc. as names.

Reply:
You are right, I don't need the Product as part of the styled-component. Sometimes we used it, but only when there were more than one Article, Container etc. I changed it